codes/quantum/qubits/surface/surface/surface.yml

```diff
@@ -134,6 +134,8 @@ realizations:
     One cycle of syndrome readout on 19-qubit planar and 24-qubit toric codes realized in two-dimensional Rydberg atomic arrays \cite{arxiv:2112.03923}.
   - |
     Signatures of corresponding topological phase of matter detected in superconducting circuits \cite{arxiv:2104.01180} and two-dimensional Rydberg atomic arrays \cite{arXiv:2104.04119}.
+  - |
```

Comment:
This has already been added in code_id: stab_4_2_2. In the future, please do a repo-wide search of the arxiv reference before you add it.